app/.gitignore

```diff
@@ -43,4 +43,7 @@ app.*.map.json
 # Android Studio will place build artifacts here
 /android/app/debug
 /android/app/profile
-/android/app/release
\ No newline at end of file
+/android/app/release
```

Comment:
Maybe this change is required for your development environment but should not be pushed to Github as it will break other people's oauth setup.